# Created at 2021-02-08T21:29:13.537
TestSet has not finished before stream error has appeared >> initializing exit by non-null configuration: DEFAULT
java.io.EOFException
	at java.base/java.io.DataInputStream.readInt(DataInputStream.java:397)
	at org.apache.maven.surefire.booter.MasterProcessCommand.decode(MasterProcessCommand.java:115)
	at org.apache.maven.surefire.booter.CommandReader$CommandRunnable.run(CommandReader.java:391)
	at java.base/java.lang.Thread.run(Thread.java:834)


# Created at 2021-02-08T21:29:19.467
Unexpected IOException: 8,1,testautomation.uiautomation.testing.AppTest,test(testautomation.uiautomation.testing.AppTest),null,Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.\000ABuild info: version: '3.141.59'\002C revision: 'e82be7d358'\002C time: '2018-11-14T08:17:03'\000ASystem info: host: 'DESKTOP-R3IULL6'\002C ip: '172.29.192.1'\002C os.name: 'Windows 10'\002C os.arch: 'amd64'\002C os.version: '10.0'\002C java.version: '11.0.8'\000ADriver info: driver.version: RemoteWebDriver,null,Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.\000ABuild info: version: '3.141.59'\002C revision: 'e82be7d358'\002C time: '2018-11-14T08:17:03'\000ASystem info: host: 'DESKTOP-R3IULL6'\002C ip: '172.29.192.1'\002C os.name: 'Windows 10'\002C os.arch: 'amd64'\002C os.version: '10.0'\002C java.version: '11.0.8'\000ADriver info: driver.version: RemoteWebDriver,AppTest.test:54 \00BB UnreachableBrowser Could not start a new session. Possible c...,org.openqa.selenium.remote.UnreachableBrowserException: \000ACould not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.\000ABuild info: version: '3.141.59'\002C revision: 'e82be7d358'\002C time: '2018-11-14T08:17:03'\000ASystem info: host: 'DESKTOP-R3IULL6'\002C ip: '172.29.192.1'\002C os.name: 'Windows 10'\002C os.arch: 'amd64'\002C os.version: '10.0'\002C java.version: '11.0.8'\000ADriver info: driver.version: RemoteWebDriver\000A\0009at testautomation.uiautomation.testing.AppTest.test(AppTest.java:54)\000ACaused by: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:4444\000A\0009at testautomation.uiautomation.testing.AppTest.test(AppTest.java:54)\000ACaused by: java.net.ConnectException: Connection refused: connect\000A\0009at testautomation.uiautomation.testing.AppTest.test(AppTest.java:54)\000A


# Created at 2021-02-08T21:29:19.470
Unexpected IOException: 8,1,testautomation.uiautomation.testing.AppTest,test(testautomation.uiautomation.testing.AppTest),null,Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.\000ABuild info: version: '3.141.59'\002C revision: 'e82be7d358'\002C time: '2018-11-14T08:17:03'\000ASystem info: host: 'DESKTOP-R3IULL6'\002C ip: '172.29.192.1'\002C os.name: 'Windows 10'\002C os.arch: 'amd64'\002C os.version: '10.0'\002C java.version: '11.0.8'\000ADriver info: driver.version: RemoteWebDriver,null,Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.\000ABuild info: version: '3.141.59'\002C revision: 'e82be7d358'\002C time: '2018-11-14T08:17:03'\000ASystem info: host: 'DESKTOP-R3IULL6'\002C ip: '172.29.192.1'\002C os.name: 'Windows 10'\002C os.arch: 'amd64'\002C os.version: '10.0'\002C java.version: '11.0.8'\000ADriver info: driver.version: RemoteWebDriver,AppTest.test:54 \00BB UnreachableBrowser Could not start a new session. Possible c...,org.openqa.selenium.remote.UnreachableBrowserException: \000ACould not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.\000ABuild info: version: '3.141.59'\002C revision: 'e82be7d358'\002C time: '2018-11-14T08:17:03'\000ASystem info: host: 'DESKTOP-R3IULL6'\002C ip: '172.29.192.1'\002C os.name: 'Windows 10'\002C os.arch: 'amd64'\002C os.version: '10.0'\002C java.version: '11.0.8'\000ADriver info: driver.version: RemoteWebDriver\000A\0009at testautomation.uiautomation.testing.AppTest.test(AppTest.java:54)\000ACaused by: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:4444\000A\0009at testautomation.uiautomation.testing.AppTest.test(AppTest.java:54)\000ACaused by: java.net.ConnectException: Connection refused: connect\000A\0009at testautomation.uiautomation.testing.AppTest.test(AppTest.java:54)\000A


# Created at 2021-02-08T21:29:19.475
Unexpected IOException: 8,1,testautomation.uiautomation.testing.AppTest,test(testautomation.uiautomation.testing.AppTest),null,java.util.concurrent.ExecutionException: java.lang.NullPointerException,null,java.util.concurrent.ExecutionException: java.lang.NullPointerException,AppTest.test \00BB TestNG java.util.concurrent.ExecutionException: java.lang.NullP...,org.testng.TestNGException: java.util.concurrent.ExecutionException: java.lang.NullPointerException\000ACaused by: java.util.concurrent.ExecutionException: java.lang.NullPointerException\000ACaused by: java.lang.NullPointerException\000A


# Created at 2021-02-08T21:29:19.570
Unexpected IOException with stream: log4j:WARN No appenders could be found for logger (freemarker.cache).


# Created at 2021-02-08T21:29:19.571
Unexpected IOException with stream: log4j:WARN Please initialize the log4j system properly.


# Created at 2021-02-08T21:29:20.115
Unexpected IOException with stream: suite Cleanup


# Created at 2021-02-08T21:29:20.328
Unexpected IOException: 2,1,org.apache.maven.surefire.testng.TestNGXmlTestSuite,TestSuite,null,null,null


